src/templates/calc.html

```diff
@@ -8,23 +8,38 @@
     form {
       max-width: 500px;
     }
+    .inline-link {
+      margin-left: 1em;
+      font-size: 0.95em;
+    }
   </style>
   <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />
 {% endblock %}
 {% block body %}
 <content class="grid">
   <form class="grid gap">
     <label for="challengers">Name:</label>
-    <select id="challengers"  class="span2" name="challengers">
-      {% for challenger in challengers %}
-        <option 
-          value="{{ challenger.bmr }}"
-          {{'selected' if name == challenger.name}}
-        >
-          {{ challenger.name }}
-        </option>
-      {% endfor %}
-    </select>
+    <div style="display: flex; align-items: center;">
```

Comment:
Applies the above class to this div instead of just using the style

```suggestion
    <div class="select-group">
```